azure-pipelines.yml

```diff
@@ -361,6 +372,54 @@ jobs:
 
     displayName: 'Test pip Install'
 
+- job: UnitTest
+  displayName: Unit Test for Core and Telemetry
+  timeoutInMinutes: 10
+
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python27:
+        python.version: '2.7'
+      Python36:
+        python.version: '3.6'
+      Python38:
+        python.version: '3.8'
+  steps:
+    - task: UsePythonVersion@0
+      displayName: 'Use Python $(python.version)'
+      inputs:
+        versionSpec: '$(python.version)'
+    - bash: pip install --upgrade pip tox
+      displayName: 'Install pip and tox'
+    - bash: ./scripts/ci/unittest.sh
+      displayName: 'Run Unit Test'
+
+- job: IntegrationTestAgainstProfiles
+  displayName: Integration Test against Profiles
+  dependsOn: BuildPythonWheel
+  condition: succeeded()
+  timeoutInMinutes: 20
+
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python36:
+        python.version: '3.6'
+      Python38:
+        python.version: '3.8'
+  steps:
+    - task: UsePythonVersion@0
+      displayName: 'Use Python $(python.version)'
+      inputs:
+        versionSpec: '$(python.version)'
+    - bash: pip install --upgrade pip wheel
+      displayName: 'Install pip and wheel'
+    - bash: ./scripts/ci/test_profile_integration.sh
```

Comment:
It looks what test_profile_integration.sh does is verifying all the modules can be loaded correctly according to [here](https://github.com/Azure/azure-cli/blob/dev/scripts/ci/test_profile_integration.sh#L23). Integration Tests against different profiles are done in job RunAutomationxxxx, right?

Reply:
Yes, need to be more accurate.  What name would you like?

Reply:
VerifyCommandsLoadingAgainstProfiles ?